packages/gestalt/src/PageHeaderComponents.js

```diff
@@ -0,0 +1,205 @@
+import IconButton from './IconButton.js';
+import Text from './Text.js';
+import Link from './Link.js';
+import Flex from './Flex.js';
+import Mask from './Mask.js';
+import Image from './Image.js';
+import Dropdown from './Dropdown.js';
+import { type ActionType } from './PageHeader.js';
+
+export function PageHeaderTitle({
+  marginTop,
+  title,
+}: {|
+  marginTop: ?number,
+  title: string,
+|}): Node {
+  return (
+    <Fragment>
+      <Box display="block" smDisplay="none">
+        <Heading size="400" lineClamp={1} accessibilityLevel={1}>
+          {title}
+        </Heading>
+      </Box>
+      <Box
+        display="none"
+        smDisplay="block"
+        dangerouslySetInlineStyle={{ __style: marginTop ? { marginTop: `${marginTop}px` } : {} }}
+      >
+        <Heading size="500" lineClamp={1} accessibilityLevel={1}>
+          {title}
+        </Heading>
+      </Box>
+    </Fragment>
+  );
+}
+
+export function PageHeaderThumbnail({ thumbnail }: {| thumbnail: Element<typeof Image> |}): Node {
+  return (
+    <Box display="none" smDisplay="block" aria-hidden>
+      <Mask height={48} rounding={2} width={48}>
+        {thumbnail}
+      </Mask>
+    </Box>
+  );
+}
+
+export function PageHeaderBadge({
+  badgeText,
+  badgeTooltipText,
+}: {|
+  badgeText: string,
+  badgeTooltipText?: string,
+|}): Node {
+  return badgeTooltipText ? (
+    <Tooltip accessibilityLabel="" text={badgeTooltipText} idealDirection="up">
+      <Badge text={badgeText} type="info" position="top" />
+    </Tooltip>
+  ) : (
+    <Badge text={badgeText} type="info" position="top" />
+  );
+}
+
+export function PageHeaderHelperIconButton({
+  accessibilityLabel,
+  accessibilityControls,
+  accessibilityExpanded,
+  onClick,
+}: {|
+  accessibilityLabel: string,
+  accessibilityControls: string,
+  accessibilityExpanded: boolean,
+  onClick: ({|
+    event:
+      | SyntheticMouseEvent<HTMLButtonElement>
+      | SyntheticKeyboardEvent<HTMLButtonElement>
+      | SyntheticMouseEvent<HTMLAnchorElement>
+      | SyntheticKeyboardEvent<HTMLAnchorElement>,
+    dangerouslyDisableOnNavigation: () => void,
+  |}) => void,
+|}): Node {
+  return (
+    <IconButton
+      accessibilityControls={accessibilityControls}
+      accessibilityExpanded={accessibilityExpanded}
+      accessibilityHaspopup
+      accessibilityLabel={accessibilityLabel}
+      bgColor="lightGray"
+      icon="question-mark"
+      iconColor="darkGray"
+      onClick={onClick}
+      size="xs"
+    />
+  );
+}
+
+export function PageHeaderSubtext({
+  subtext,
+  helperLink,
+}: {|
+  subtext: string,
+  helperLink?: {|
+    text: string,
+    accessibilityLabel: string,
+    href: string,
+    onClick: ({|
+      event: SyntheticMouseEvent<HTMLAnchorElement> | SyntheticKeyboardEvent<HTMLAnchorElement>,
+      dangerouslyDisableOnNavigation: () => void,
+    |}) => void,
+  |},
+|}): Node {
+  return (
+    <Box display="none" smDisplay="block">
+      <Text inline>
+        {subtext}{' '}
```

Comment:
![image](https://user-images.githubusercontent.com/10593890/161650060-9997c0cc-b174-4074-83b4-d111054733ad.png)

 vs 

![Screen Shot 2022-04-04 at 7 45 42 PM](https://user-images.githubusercontent.com/10593890/161650291-fd05d704-6f4f-427f-a5a1-8f1f65dfed49.png)

The wrap is different using Flex. It's not wrapping as a single line of Text.

Reply:
Ah I see what you're saying 👍